### PR TITLE
Align the design of 2nd passphrase inputs in all transaction types - Closes #2198

### LINKED ISSUE #2198: Align the design of 2nd passphrase inputs in all transaction types
Align the design of 2nd passphrase inputs in all transaction types

### PR DESCRIPTION
**Pull request by osvaldovega**
### What issue have I solved?

-- #2198

### How have I implemented/fixed it?

Update css design for the component it self, this means that margin and paddings were fix only for this component and not for other elements around it.

it is almost impossible to make this element looks exactly the same as the design (perfectpixel tool) as the grid system is different so what is done here is use the same values as the design to make it looks like the design.

**NOTE:** The last week after talked with Dom about the design he change the design for what use the icon for show and hide buttons are below the inputs and not in top, you can check this screen.

https://projects.invisionapp.com/d/main/#/console/17570736/369060438/preview

### How has this been tested?

Check the component in login, send tx and delegate registration, using a second passphrase account.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

**Comment by osvaldovega** [comment]
Hi @Efefefef sorry for the confusion the design since last week change, Dom updated but he just updated some screen not all of them, here you can find it.

https://projects.invisionapp.com/d/main/#/console/17570736/369060438/preview

As you can see, this is the design that I am follow after talk with him.

**Comment by Efefefef** [comment]
@osvaldovega Alright man, then 
Margin between 'show passphrase' and 'send'
Show Passphrase with capitals
Tooltip near 'Second passphrase' ?

**Comment by osvaldovega** [comment]
Thanks @Efefefef I made the changes and should be good, please let me know

File: src/components/delegateRegistration/summary/summary.js [hunk 1]
```diff
@@ -1,28 +1,79 @@
 import React from 'react';
 import AccountVisual from '../../accountVisual/index';
 import { fromRawLsk } from '../../../utils/lsk';
+import PassphraseInputV2 from '../../passphraseInputV2/passphraseInputV2';
+import { PrimaryButtonV2, TertiaryButtonV2 } from '../../toolbox/buttons/button';
+import { extractPublicKey } from '../../../utils/account';
 import Fees from '../../../constants/fees';
-import TransactionSummary from '../../transactionSummary';
```
Comment (yasharAyari) [inline src/components/delegateRegistration/summary/summary.js]: Why did you remove TransactionSummary from this delegate registration summary component? delegate registration is a transaction type and it should use the general TransactionSummary component as we talk about it.

Comment (osvaldovega) [reply src/components/delegateRegistration/summary/summary.js]: that was causing a problem related the second passphrase, because of that component the second passphrase was being displayed in the middle of the component above of Fee.

**Comment by osvaldovega** [comment]
done @Efefefef I aligned the summary for send component too and fix merge conflicts

File: src/components/delegateRegistration/summary/summary.js (hunk 1 above)
Comment (yasharAyari) [reply src/components/delegateRegistration/summary/summary.js]: Screenshot 2019-07-08 at 12 27 13
the design of the delegate summary isn’t aligned with the latest design. The fee should always be at the top of second passphrase inputs. So, please change the transaction summary to show the fee at the top of second passphrase input and use that instead of your custom summary component.

Comment (osvaldovega) [reply src/components/delegateRegistration/summary/summary.js]: Yashar I think is better to refactor the TransationSummary component, right now the delegate Summary is exactly the same as the Send Summary

after refactor the TransactionSummary component to looks exactly as should be in all pages I can replace it in Send/Summary and DelegateReg/Summarywill be easier

Comment (yasharAyari) [reply src/components/delegateRegistration/summary/summary.js]: It doesn't require refactoring at all. You just need to rearrange some elements inside the render method to be aligned with the design. You must move the fee section to the top of the passphrase input component.

**Comment by osvaldovega** [comment]
Requested changes done.
@Efefefef please review the PR again
